block/manager.go

```diff
@@ -482,13 +519,8 @@ func (m *Manager) exponentialBackoff(backoff time.Duration) time.Duration {
 }
 
 // TODO(tzdybal): consider inlining
-func (m *Manager) publishHeader(block *types.Block) {
-	m.HeaderOutCh <- &block.Header
-}
-
-// TODO(tzdybal): consider inlining
-func (m *Manager) publishCommit(commit *types.Commit) {
-	m.CommitOutCh <- commit
+func (m *Manager) publishSignedHeader(block *types.Block, commit *types.Commit) {
+	m.HeaderOutCh <- &types.SignedHeader{Header: block.Header, Commit: *commit}
 }
```

Comment:
If we now combine publishing with one HeaderOutCh, do we use CommitOutCh elsewhere?

Reply:
Good catch! I missed this while refactoring.